src/frontend/js/utils/test/factories.ts

```diff
@@ -176,7 +181,9 @@ export const JoanieEnrollmentFactory = createSpec({
 export const TargetCourseFactory = createSpec({
   code: faker.unique(faker.random.alphaNumeric(5)),
   organizations: [],
-  title: faker.random.words(1, 3),
+  title: derived(({ code }: { code: string }) => {
+    return `${faker.random.words(1, 3)()}(${code})`;
+  }),
```

Comment:
What is the purpose ?

Reply:
It's explained on the commit message: 
```
Sometimes multiple TargetCourse.title were identical.
We need unique title to test that all targetCourses are displayed.
```

I had a flaky test trying to get a title that appear multiple time in the component.
It came from this `faker.random.words(1, 3)` that sometime give twice the same title on different `TargetCourse`

Reply:
You could use `faker.unique(faker.random.words(1, 3))`